static/app/views/starfish/modules/APIModule/APIModuleView.tsx

```diff
@@ -202,22 +202,12 @@ export default function APIModuleView({location, onSelect}: Props) {
       interval
     );
 
-  const tpmTransactionSeries = queryToSeries(
-    topTransactionsData,
-    'group',
-    'epm()',
-    startTime,
-    endTime,
-    24
-  );
+  const tpmTransactionSeries = queryToSeries(topTransactionsData, 'group', 'epm()');
 
   const p75TransactionSeries = queryToSeries(
     topTransactionsData,
     'group',
-    'p75(transaction.duration)',
-    startTime,
-    endTime,
-    24
```

Comment:
👍 we're removing these because the data returned back should already be filtered on the time now?

Reply:
So the start and end time and interval (24) here is used for determining the intervals in the series to zerofill. Since the data from discover is already zerofilled, we don't actually need to do this post processing zerofill, so we can remove these args to prevent that from happening

Reply:
I see, thanks for the explanation!